generator/src/main/kotlin/com/android/gradle/replicator/generator/project/GradleProjectGenerator.kt

```diff
@@ -241,7 +250,7 @@ class GradleProjectGenerator(
         val generationPropertyFile = folder.join("generation.properties")
 
         val loader = Thread.currentThread().contextClassLoader
-        loader.getResourceAsStream("project/resourceGeneratorConstants.properties")!!.copyTo(FileOutputStream(generationPropertyFile))
+        loader.getResourceAsStream("project/resource-generator-constants.properties")!!.copyTo(FileOutputStream(generationPropertyFile))
```

Comment:
using getResourceAsStream will leak the jar file and class loader if not closed properly